packages/starknet-snap/test/src/executeTxn.test.ts

```diff
@@ -140,11 +197,12 @@ describe('Test function: executeTxn', function () {
         },
       ],
       undefined,
-      { maxFee: 100 },
+      { maxFee: '22702500105945', nonce: 1 },
     );
   });
 
   it('should throw error if executeTxn fail', async function () {
+    sandbox.stub(utils, 'isAccountDeployed').resolves(false);
```

Comment:
```suggestion
    sandbox.stub(utils, 'isAccountDeployed').resolves(true);
```
make it deployed to bypass the mock of  the create account, as we have already verify from above test case